pypyr/cli.py

```diff
@@ -35,6 +35,9 @@ def get_parser():
                         help='Integer log level. Defaults to 20 (INFO). '
                         '10=DEBUG\n20=INFO\n30=WARNING\n40=ERROR\n50=CRITICAL'
                         '.\n Log Level < 10 gives full traceback on errors.')
+    parser.add_argument('--logfile', dest='log_file', type=str,
```

Comment:
- rename `--logfile` to `--logpath` (in the future there might be logging providers that aren't files)
- rename `log_file` to `log_path`
- remove `type=str` - this is redundant for str types

Reply:
* rename --logfile to --logpath (in the future there might be logging providers that aren't files)
* remove type=str - this is redundant for str types